### ISSUE TITLE
Frontend: add a comma separator for metrics

### ISSUE THREAD

**Issue by qu8n** [qu8n/focusbeacon#17]
Right now, numeric metrics don't have a comma separator. For example: `1056` sessions (vs. `1,056` sessions). We'll go with the comma separator instead of period since most Focusmate users are based in the US.

**Comment by Shiv-2712** [comment]
hi, can I work on this issue?

**Comment by qu8n** [comment]
@Shiv-2712 Absolutely. You can find the components used to display metrics inside `src/components/dashboard`. I recommend calling `toLocaleString()` to produce this effect. When you're done, please push your work to a PR for my review. Happy to answer any questions at all during this process.

**Comment by qu8n** [comment]
Resolved via https://github.com/qu8n/focusbeacon/commit/f7d559e8c2a6ad91c497b320e179ee80bc413275
